pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -367,6 +369,8 @@ func (ds *Plugin) DeleteAttestedNode(ctx context.Context, spiffeID string) (atte
 		if err != nil {
 			return err
 		}
+		// TODO: this is at the wrong level of the software stack.
```

Comment:
I think we missed removal of this TODO. Can you please remove?

Reply:
Fixes in next push.